streams/stable.json

```diff
@@ -96,88 +96,92 @@
                 "aws": {
                     "regions": {
                         "af-south-1": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-09a56ca65e2be3f44"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0c449e8c47b84127c"
                         },
                         "ap-east-1": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-043a1d3f96a5d70ad"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0102d3a53442f599b"
                         },
                         "ap-northeast-1": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-097dd70e4c739653a"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-014ab2a6581db448a"
                         },
                         "ap-northeast-2": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-0fb001902730d6d1f"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0f38422bedc9f94b0"
                         },
                         "ap-northeast-3": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-072fef2180d1c4022"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-01368a584304fa692"
                         },
                         "ap-south-1": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-00ac7ea8efa83ca81"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0791420799b2c704e"
                         },
                         "ap-southeast-1": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-001d8b81fde614084"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-066cfb35d24c64435"
                         },
                         "ap-southeast-2": {
-                            "release": "35.20211203.3.0",
-                            "image": "ami-0a349e727bb67af85"
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0f392e2c8502a65e3"
+                        },
+                        "ap-southeast-3": {
+                            "release": "35.20211215.3.0",
+                            "image": "ami-0bdf3383017a731a2"
```

Comment:
note the new region.. cc @bgilbert 

xref: https://pagure.io/fedora-infrastructure/issue/10432